lint-tasks.yml

```diff
@@ -1,18 +1,22 @@
 version: "3"
 
 vars:
+  NODEJS_18_BIN_DIR: "{{.TASKFILE_DIR}}/build/nodejs_18/node/bin"
```

Comment:
Why not the (current) latest version? To maximize compatibility?

Reply:
Presumably we will periodically update the version, so perhaps we should name the variable agnostically like `LINTER_NODEJS_BIN_DIR`?